astropy/io/misc/asdf/tags/transform/tests/test_transform.py

```diff
@@ -64,11 +46,6 @@ def custom_and_analytical_inverse():
 
 test_models.extend(math_models)
 
-test_models_with_constraints = [astmodels.Legendre2D(x_degree=1, y_degree=1,
```

Comment:
I don't see any other test in here that includes `fixed` and `bounds`, can we retain coverage of that case?

Reply:
Yup, didn't mean to delete that.